lib/ui/src/containers/menu.tsx

```diff
@@ -3,7 +3,7 @@ import React, { useMemo } from 'react';
 import { Badge } from '@storybook/components';
 import { API } from '@storybook/api';
 
-import { shortcutToHumanString } from '../libs/shortcut';
+import { shortcutToHumanString } from '@storybook/api/dist/lib/shortcut';
```

Comment:
we shouldn't be importing from `dist`. can you export these from `@storybook/api/shortcut` maybe?

Reply:
To be able use the `@storybook/api/shortcut` import I also need add the `shortcut` module declaration in `lib/api/src/typings.d.ts`. And `@storybook/api/shortcut` won't be typed and won't make auto-suggestions while coding.

As an alternative I can export `shortcut` from `lib/api/src/index.tsx`:

```javascript
import * as shortcut from './lib/shortcut';

export { shortcut };
```

And use it like

```javascript
import { shortcut } from '@storybook/api';
const { shortcutToHumanString } = shortcut;
```

Which way is preferred to make `shortcut` importable?

Reply:
You can also add typings for the `shortcut` entry. Here's an example of that:

https://github.com/storybookjs/storybook/blob/next/lib/components/html.js
https://github.com/storybookjs/storybook/blob/next/lib/components/html.d.ts

Reply:
Thank you this example, fixed